lib/x11utils.pm

```diff
@@ -92,6 +92,7 @@ sub ensure_unlocked_desktop {
         }
         wait_still_screen 2;       # slow down loop
         die 'ensure_unlocked_desktop repeated too much. Check for X-server crash.' if ($counter eq 1);    # die loop when generic-desktop not matched
+        record_info('Desktop unlock attempt', "Count: $counter");  # mark attempts to check for unlocked desktop
```

Comment:
What's about `10 - $counter` ? Of course, we need to put 10 to some variable, that it doesn't appear to be magic number. Or simply revert logic to increase counter.

Reply:
I like this idea more :) Let's just revert the logic